calendar@deeppradhan/files/calendar@deeppradhan/desklet.js

```diff
@@ -0,0 +1,252 @@
+const Desklet = imports.ui.desklet;
+const Gio = imports.gi.Gio;
+const Lang = imports.lang;
+const Mainloop = imports.mainloop;
+const Settings = imports.ui.settings;
+const St = imports.gi.St;
+const Tooltips = imports.ui.tooltips;
+
+const Clutter = imports.gi.Clutter;
+const GdkPixbuf = imports.gi.GdkPixbuf;
+const Cogl = imports.gi.Cogl;
+
+const UUID = "calendar@deeppradhan";
+
+const DESKLET_DIR = imports.ui.deskletManager.deskletMeta[UUID].path;
+
+imports.searchPath.push(DESKLET_DIR);
+
+const Calendar = imports.calendar;
```

Comment:
This won't work if somebody already has a calendar.js file loaded via the same method. To future proof this, you can use this here:

```js
let Calendar;
if (typeof require !== 'undefined') {
	Calendar = require('./calendar');
} else {
	Calender = imports.ui.deskletManager.desklets[UUID].calendar;
}
```

That will work on the upcoming release as well as stay backwards compatible.